index.html

```diff
@@ -27,7 +27,7 @@
     <div id="display">
```

Comment:
if you are not using the id specified here then you do not need to have one. You may remove this id

Reply:
id=display is used in CSS for styling purpose

Reply:
unused id removed!